src/cobra/io/json.py

```diff
@@ -1,16 +1,11 @@
 """Provide functions for I/O in JSON format."""
 
+import json
 from pathlib import Path
 from typing import IO, TYPE_CHECKING, Any, Union
 
 from .dict import model_from_dict, model_to_dict
 
-
```

Comment:
You need an extra empty line here to make isort happy.

Reply:
thank you, you're making me happy too